.github/workflows/test_dbt_models.yaml

```diff
@@ -31,14 +31,16 @@ jobs:
         run: |
           python3 scripts/run_iasworld_data_tests.py \
             --target "$TARGET" \
-            --output-dir ./qc_test_results/
+            --output-dir ./qc_test_results/ \
+            --vars "{\"data_test_iasworld_commercial_sns_topic\": \"$COMMERCIAL_SNS_TOPIC\"}"
```

Comment:
One downside of this approach (setting topic ARNs as vars and overriding them with secrets during workflow execution) is that we'll have to update this bloated JSON dict every time we add a new topic. I don't think it's bad enough to outweigh the benefits of this approach, but it's a downside for sure.

Reply:
**suggestion (non-blocking):** Why not just make a jq call or similar that maps the dbt vars to their expected env var equivalent, such that if we add a new dbt var it will automatically search for the corresponding env var? E.g.

- Add `data_test_iasworld_commercial_sns_topic` searches for `$COMMERCIAL_SNS_TOPIC`
- Add `data_test_iasworld_asmt_sns_topic` searches for `$ASMT_SNS_TOPIC`

Reply:
Hmm, this is an interesting idea, but I'm not totally convinced it's worth the complexity just yet. Let's plan to discuss in-person tomorrow.

Reply:
Based on our in-person discussion, I'm going to table this until we need to add more topics.